src/main/webapp/app/exercises/shared/result/result-detail.component.ts

```diff
@@ -129,4 +217,104 @@ export class ResultDetailComponent implements OnInit {
             }),
         );
     };
+
+    /**
+     * Filters / Summarizes positive test cases for a student and programming exercise result
+     * @param feedbackList The list of feedback items
+     * @private
+     */
+    private filterFeedbackItems(feedbackList: FeedbackItem[]) {
+        if (this.exerciseType !== ExerciseType.PROGRAMMING || this.showTestDetails) {
+            return [...feedbackList];
+        } else {
+            const positiveTestCases = feedbackList.filter((feedbackItem) => {
+                return feedbackItem.type === FeedbackItemType.Test && feedbackItem.positive;
+            });
+            if (positiveTestCases.length > 0) {
+                return [
+                    {
+                        type: FeedbackItemType.Test,
+                        category: 'Feedback',
+                        title: positiveTestCases.length + ' passed test' + (positiveTestCases.length > 1 ? 's' : ''),
+                        positive: true,
+                        credits: positiveTestCases.reduce((sum, feedbackItem) => sum + (feedbackItem.credits || 0), 0),
+                    },
+                    ...feedbackList.filter((feedbackItem) => !positiveTestCases.includes(feedbackItem)),
+                ];
+            } else {
+                return [...feedbackList];
+            }
+        }
+    }
+
+    /**
+     * Handles the coloring of each feedback items based on its type and credits.
+     * @param feedback The feedback item
+     */
+    getClassNameForFeedbackItem(feedback: FeedbackItem): string {
+        if (feedback.type === FeedbackItemType.Issue) {
+            return 'alert-warning';
```

Comment:
Now we have both the SCA and the manual feedback with score 0 in yellow. I think this is okay for now. If we encounter some misunderstandings from the students side, we can adjust it in a follow up  :)